tests/audioworklet/shell.html

```diff
@@ -0,0 +1,1305 @@
+              element.value += text + "\n";
+              element.scrollTop = element.scrollHeight; // focus on bottom
+            }
+          };
+        })(),
+        printErr: function(text) {
+          if (arguments.length > 1) text = Array.prototype.slice.call(arguments).join(' ');
+          console.error(text);
+        },
+        canvas: (function() {
+          var canvas = document.getElementById('canvas');
+
+          // As a default initial behavior, pop up an alert when webgl context is lost. To make your
+          // application robust, you may want to override this behavior before shipping!
+          // See http://www.khronos.org/registry/webgl/specs/latest/1.0/#5.15.2
+          canvas.addEventListener("webglcontextlost", function(e) { alert('WebGL context lost. You will need to reload the page.'); e.preventDefault(); }, false);
+
+          return canvas;
+        })(),
+        setStatus: function(text) {
+          if (!SampleInstance.setStatus.last) SampleInstance.setStatus.last = { time: Date.now(), text: '' };
+          if (text === SampleInstance.setStatus.last.text) return;
+          var m = text.match(/([^(]+)\((\d+(\.\d+)?)\/(\d+)\)/);
+          var now = Date.now();
+          if (m && now - SampleInstance.setStatus.last.time < 30) return; // if this is a progress update, skip it if too soon
+          SampleInstance.setStatus.last.time = now;
+          SampleInstance.setStatus.last.text = text;
+          if (m) {
+            text = m[1];
+            progressElement.value = parseInt(m[2])*100;
+            progressElement.max = parseInt(m[4])*100;
+            progressElement.hidden = false;
+            spinnerElement.hidden = false;
+          } else {
+            progressElement.value = null;
+            progressElement.max = null;
+            progressElement.hidden = true;
+            if (!text) spinnerElement.style.display = 'none';
+          }
+          statusElement.innerHTML = text;
+        },
+        totalDependencies: 0,
+        monitorRunDependencies: function(left) {
+          this.totalDependencies = Math.max(this.totalDependencies, left);
+          SampleInstance.setStatus(left ? 'Preparing... (' + (this.totalDependencies-left) + '/' + this.totalDependencies + ')' : 'All downloads complete.');
+        }
+      };
+      SampleInstance.setStatus('Downloading...');
+      window.onerror = function(event) {
+        // TODO: do not warn on ok events like simulating an infinite loop or exitStatus
+        SampleInstance.setStatus('Exception thrown, see JavaScript console');
+        spinnerElement.style.display = 'none';
+        SampleInstance.setStatus = function(text) {
+          if (text) SampleInstance.printErr('[post-exception status] ' + text);
+        };
+      };
+    </script>
+    {{{ SCRIPT }}}
+    <script>
+      window.addEventListener("load", function() {
+        AudioWorkletSample(SampleInstance);
+      });
+    </script>
+  </body>
+</html>
+

```

Comment:
Do we need to check in this huge html file?   Where does it come from?

Reply:
It's just a copy of `src/shell.html` customized with the export name because:
```
emcc: error: pthreads + MODULARIZE currently require you to set -s EXPORT_NAME=Something (see settings.js) to Something != Module, so that the .worker.js file can work
```

And then setting EXPORT_NAME
```
emcc: error: Customizing EXPORT_NAME requires that the HTML be customized to use that name (see https://github.com/emscripten-core/emscripten/issues/10086)
```

Actually, if I make it run with MINIMAL_RUNTIME I can get rid of this... hold on :)

Reply:
Couldn't make it run with MINIMAL_RUNTIME so far, but I did find that other tests use `test_file('shell_that_launches_modularize.html')` that loads `MyModule` so I just used that as well and renamed my module to MyModule! 

So this removes the entire html.